zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/job/ActivatableJobsPushTest.java

```diff
@@ -165,9 +171,9 @@ public void shouldPushWhenJobTimesOut() {
     jobRecords(TIMED_OUT).withType(jobType).await();
 
     // then
-    assertJobActivations(activationCount);
-    assertEventOrder(JobIntent.TIME_OUT, JobIntent.TIMED_OUT, JobBatchIntent.ACTIVATED);
-    assertActivatedJob(jobKey, activationCount);
+    // expect that the job is re-activated after job was timed out
+    assertEventOrder(JOB_AND_JOB_BATCH_TYPES, JobIntent.TIMED_OUT, JobBatchIntent.ACTIVATED);
```

Comment:
💭 Rather than a comment, it's also possible to add a description String to the `assertEventOrder` method's parameters for use in the actual assertion

Reply:
Done!